processing/src/main/java/io/druid/query/filter/Filter.java

```diff
@@ -36,8 +42,15 @@
    *
    * @see Filter#estimateSelectivity(BitmapIndexSelector)
    */
-  ImmutableBitmap getBitmapIndex(BitmapIndexSelector selector);
+  default ImmutableBitmap getBitmapIndex(BitmapIndexSelector selector)
+  {
+    return getBitmapResult(selector, new DefaultBitmapResultFactory(selector.getBitmapFactory()));
+  }
 
+  default <T> T getBitmapResult(BitmapIndexSelector selector, BitmapResultFactory<T> bitmapResultFactory)
```

Comment:
Is it appropriate to mark this one `@Deprecated` so that any usage gets the notice about the intended 0.11 change?